src/main/java/edu/wpi/first/gradlerio/GradleRIOPlugin.java

```diff
@@ -77,6 +78,18 @@ public void execute(Parameters parameters) {
 
     @Override
     public void apply(Project project) {
+        boolean onedrive = false;
+        try {
+            if (project.getRootDir().toString().toUpperCase().contains("ONEDRIVE")) {
```

Comment:
You don't need the if here. You can just assign onedrive straight to the whole result of the contains statement.